pymc3/step_methods/hmc/quadpotential.py

```diff
@@ -73,15 +71,18 @@ def __str__(self):
 
 
 class QuadPotential(object):
-    def velocity(self, x):
+    def velocity(self, x, out=None):
```

Comment:
yes. Since we are doing this in python now, avoiding allocations seems to matter.